tikv-client/src/main/java/com/pingcap/tikv/predicates/TiKVScanAnalyzer.java

```diff
@@ -30,10 +30,7 @@
 import com.pingcap.tikv.expression.Expression;
 import com.pingcap.tikv.expression.PartitionPruner;
 import com.pingcap.tikv.expression.visitor.IndexMatcher;
-import com.pingcap.tikv.key.IndexScanKeyRangeBuilder;
-import com.pingcap.tikv.key.Key;
-import com.pingcap.tikv.key.RowKey;
-import com.pingcap.tikv.key.TypedKey;
+import com.pingcap.tikv.key.*;
```

Comment:
Disable wild import
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports